tests/unit/forms/rsvp_json_test.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+from funnel import forms
+
+valid_json = json.dumps(
+    '''{
+            "fields": [    {
+            "description": "An explanation for this field",
+            "name": "field_name","title": "Field label shown to user",
+            "type": "string"},{"name": "has_checked",
+            "title": "I accept the terms","type": "boolean"},
+            {"choices":
+            ["First choice","Second choice","Third choice"],
+            "name": "choice","title": "Choose one"}]
+            }'''
+)
+
+rsvp_excess_json = {
+    'choice': 'First choice',
+    'field_name': 'Twoflower',
+    'has_checked': 'on',
+    'company': 'MAANG',
+}
+
+form_valid_json = {
+    "fields": [
+        {
+            "description": "An explanation for this field",
+            "name": "field_name",
+            "title": "Field label shown to user",
+            "type": "string",
+        },
+        {"name": "has_checked", "title": "I accept the terms", "type": "boolean"},
+        {
+            "choices": ["First choice", "Second choice", "Third choice"],
+            "name": "choice",
+            "title": "Choose one",
+        },
+    ]
+}
+
+
+def test_valid_boxoffice_form(app) -> None:
+    with app.test_request_context(
+        method='POST',
+        data={'register_form_schema': valid_json},
+    ):
+        form = forms.ProjectBoxofficeForm(meta={'csrf': False})
+        assert form.validate() is True
```

Comment:
This test is broken. A string was submitted and accepted as valid JSON, and the form's validator did not notice that it is not in the correct format.